src/Queue.php

```diff
@@ -56,6 +56,10 @@
     const STATUS_DONE = 3;
 
     /**
+     * @var bool
```

Comment:
needs a description and `@since` annotation.

Reply:
I still don't know version that will include the PR.